src/ui/popup/theme/controls/mode.less

```diff
@@ -0,0 +1,27 @@
+@import "../../../theme";
+
+.static-edit-button {
+    background-color: @color-heading;
+    border-radius: @size-text-small-height / 2;
+    color: @color-control-fore;
+    cursor: pointer;
+    font-size: @size-text-small;
+    line-height: @size-text-small-height;
+    left: 35%;
```

Comment:
This button is not related to the whole ControlGroup, but only to the control itself. So, the `position: relative` should be assigned to a container of both control and Edit button over it. `left` should be relative to the left edge of the control.